src/components/Confirm/index.js

```diff
@@ -194,7 +194,7 @@ class Confirm extends Component  {
     }
     else if  (method === 'face') {
       if (variant === 'video') {
-        const data = { challengeData, blob }
+        const data = { challengeData, blob, language: this.props.i18n.currentLocale }
```

Comment:
[Minor] You have props destructed at the top of this method, so this could be put there too (just to keep consistency).